pnpm-lock.yaml

```diff
@@ -1,8 +1,9 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
I've used pnpm v6.35.1 to install `@ember/string`, which is the same version used in CI. This means that this lock file was generated using a newer version, so pnpm downgrades it.